js/packages/teams-ai/src/prompts/ActionOutputMessage.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @module teams-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TurnContext } from 'botbuilder';
+
+import { Memory } from '../MemoryFork';
+import { Tokenizer } from '../tokenizers';
+import { Message } from './Message';
+import { PromptFunctions } from './PromptFunctions';
+import { RenderedPromptSection } from './PromptSection';
+import { PromptSectionBase } from './PromptSectionBase';
+
+/**
+ * A section capable of rendering user input text and images as a user message.
+ */
+export class ActionOutputMessage extends PromptSectionBase {
+    private readonly _inputVariable: string;
+
+    /**
+     * Creates a new 'ActionOutputMessage' instance.
+     * @param {number} tokens Optional. Sizing strategy for this section. Defaults to `auto`.
+     * @param {string} inputVariable Optional. Name of the variable containing the user input text. Defaults to `input`.
+     */
+    public constructor(tokens: number = -1, inputVariable = 'actionOutputs') {
+        super(tokens, true, '\n', 'action: ');
```

Comment:
do we need an "action" prefix?

Reply:
I believe in a call @aacebo said no need but maybe I'm misremembering. @aacebo thoughts?

Reply:
Keeping for `renderAsText`